chef/cookbooks/nova/templates/default/nova.conf.erb

```diff
@@ -2779,6 +2788,9 @@ connection=<%= @database_connection %>
 # (integer value)
 # Deprecated group/name - [DATABASE]/sqlalchemy_pool_timeout
 #pool_timeout=<None>
+<% unless node[:nova][:db][:pool_timeout].nil? -%>
+pool_timeout=<%= node[:nova][:db][:pool_timeout] %>
```

Comment:
Personally I prefer a clean backport and the more verbose variant, since it is easier to grep (grep "^foobar=" gives a hit).